!Command
command:
- getSeq_genome_wN
positional: []
named: []
parent:
subcommands: []
usage: []
help_flag:
usage_flag:
version_flag:
help_text: "Traceback (most recent call last):\n  File \"/usr/local/bin/getSeq_genome_wN\"\
  , line 10, in <module>\n    sys.exit(main())\n  File \"/usr/local/lib/python3.9/site-packages/ptools_bin/getSeq_genome_wN.py\"\
  , line 95, in main\n    with open(sys.argv[1], \"rt\") as f:\nFileNotFoundError:\
  \ [Errno 2] No such file or directory: '--help'\n"
generated_using:
- --help
docker_image: quay.io/biocontainers/ptools_bin:0.0.5--pyh3252c3a_0
